XCUITests/TopTabsTest.swift

```diff
@@ -24,7 +24,7 @@ class TopTabsTest: BaseTestCase {
         waitUntilPageLoad()
         waitForTabsButton()
         // The tabs counter shows the correct number
-        waitForExistence(app.buttons["Show Tabs"])
+        waitForExistence(app.buttons["Show Tabs"], timeout: 10)
```

Comment:
This `waitForExistence is redundant`, this is already checked in the previous statement, please remove from here...